### DISCUSSION TITLE
Add QD Field for Disqualified Times

### DISCUSSION THREAD

**Discussion by jt-prop** [JohnStrunk/wahoo-results#380] [Ideas & feature requests]
Love the scoreboard display. It is the only scoreboard that works with Dolphin system. Like to request and see if it is possible to add QD field as part of the results. I notice that the information is not written in the .do3 or .do4 files, but exists in the .txt, .xml and .csv files.

**Comment by JohnStrunk** [comment]
Thanks for the feedback! I'm glad it's useful for you.

Adding support for the other file formats is a pretty sizable job.

We've never used either the DQ or Empty fields in the Dolphin software and just handled everything through MM.
I'm curious how you'd want the DQs represented? Would it be something like replacing the final time field on the board w/ "DQ"?

**Comment by jt-prop** [comment]
Hi John,
The reason for flagging DQ is to let the swimmers/coach know of any disqualification real time. Refs don't always talk to swimmers after they been disqualified and also gives the opportunity for coaches to talk to their swimmers or dispute any DQ time. A simple "D" next to the time would suffice. Also, any DQ would not place.
Thanks,

-Juan Torres

 

    On Wednesday, September 6, 2023 at 05:13:58 AM PDT, John Strunk ***@***.***> wrote:  
 
 


Thanks for the feedback! I'm glad it's useful for you.

Adding support for the other file formats is a pretty sizable job.

We've never used either the DQ or Empty fields in the Dolphin software and just handled everything through MM.
I'm curious how you'd want the DQs represented? Would it be something like replacing the final time field on the board w/ "DQ"?

**Comment by jt-prop** [comment]
We have a designated CTS operator for the Dolphin, and they enter any DQ realtime which would show up on a scoreboard. They also have the option to empty any lanes.

-Juan Torres

 

    On Wednesday, September 6, 2023 at 08:17:38 AM PDT, ***@***.*** ***@***.***> wrote:  
 
 Hi John,
The reason for flagging DQ is to let the swimmers/coach know of any disqualification real time. Refs don't always talk to swimmers after they been disqualified and also gives the opportunity for coaches to talk to their swimmers or dispute any DQ time. A simple "D" next to the time would suffice. Also, any DQ would not place.
Thanks,

-Juan Torres

 

    On Wednesday, September 6, 2023 at 05:13:58 AM PDT, John Strunk ***@***.***> wrote:  
 
 


Thanks for the feedback! I'm glad it's useful for you.

Adding support for the other file formats is a pretty sizable job.

We've never used either the DQ or Empty fields in the Dolphin software and just handled everything through MM.
I'm curious how you'd want the DQs represented? Would it be something like replacing the final time field on the board w/ "DQ"?

**Comment by JohnStrunk** [comment]
#818 Adds the ability to show DQs on the scoreboard. Options are:
- Ignore: DQs are not shown (the default, existing behavior)
- DQs are shown in the "place" column, and the time is displayed as usual
- DQs are shown in the time column, hiding the swimmer's time

CSV format parsing has also been added to get the DQ information. This should be in the next release.
